lib/OpenQA/Worker/Cache/Client.pm

```diff
@@ -108,6 +108,14 @@ sub asset_exists { !!(-e shift->asset_path(@_)) }
 
 sub request { OpenQA::Worker::Cache::Request->new(client => shift) }
 
+sub availability_error {
```

Comment:
Because this method is not supposed to catch any other errors than the cache being not available. If other things go wrong (which is likely possible) this method does not reflect that.